Day to day work/Project documentation.md

```diff
@@ -0,0 +1,43 @@
+## Project Documentation
+
+The documentation should be separated into a few levels of the project you are working on. That way, anyone can easily find the info they are looking for by searching specific files section.
+
+Those levels are shown in the following diagram:
+![](/img/project_documentation/documentation_chart.png)
+
+### Base documentation - readme
+The readme of the platform (i.e., Android, iOS, Flutter, etc.) should contain only chapters defined in the [mobile template readme](https://github.com/infinum/mobile-readme-template). The idea behind that readme is to have only crucial information regarding the project's platform.
+
+You can find the template here: [Mobile readme template](https://github.com/infinum/mobile-readme-template)
+
+### Platform wiki
+If you need to add anything extra that is not covered by the readme template, you should create a wiki (e.g., repo wiki on GitHub). 
+
+That wiki can contain topics such as:
+- Security storage usage
+- Project Specifics (e.g., Xcode, Android Studio)
+- Third-party integrations
+- Testing Setup
+- Pentest preparation
+- etc.
+
+This wiki should only contain platform-specific documentation.
+
+### Project documentation
+In general, this documentation should be the largest one. Anything related to the project should be put in here. If you have to write the same thing on both readmes or platform wikis, that should be done here.
+
+Some of the examples:
+- General project info
+- Onboarding & setup
+- App activation
+- VPN setup
+- Illustrations and icon naming
+- Feature flags
+- Test users
+- etc.
+
+Anyone working on the project (i.e., Developers, Designers, QA, PM, etc.) should participate in creating this document. 
+This is the only documentation that is not defined by technology. So feel free to write it on a separate Wiki (GitHub) repo, Confluence, Productive, Google Docs, etc.
+
```

Comment:
I would maybe add here a subsection about feature level documentation. And add few examples:
* [Feature flags on HPB](https://github.com/infinum/documentation-hpb-mhpb/wiki/Features-flag)
* [Light group commissioning - Add device to Zigbee a distributed network-v5-20231003_150804.pdf](https://github.com/infinum/devproc-handbook/files/12842969/Light.group.commissioning.-.Add.device.to.Zigbee.a.distributed.network-v5-20231003_150804.pdf) cc @goranbrl 
* [A1 Component specs](https://github.com/infinum/Specification-Moj-A1-API/blob/master/components.md)

**A note to ourselves:** Check if this chapter and handbook is public, because we probably do not want to have these examples public because of contracts with the clients. However it would be nice to show off with these examples to the public :D

Reply:
This PR is opened on the public part of the handbook. I thoguh the same -> "However it would be nice to show off with these examples to the public". 

If we create example project, then this can be public without the problem :) 

P.S. Can you please remove links to the documents as this is the _public_ repo